docs/source/notebooks/AutoEncoders.md

```diff
@@ -73,99 +75,91 @@ print(x_test.shape)
 
 ## 3. Define Keras Model
 
-We will be defining a very simple autencoder. We define _three_ model building methods:
+We will be defining a very simple autencoder. We define _three_ models:
 
-1. One to build a full end-to-end autoencoder.
-2. One to create a model that includes only the encoder portion.
-3. One that creates a model that includes only the decoder portion.
+1. An encoder
+2. A decoder
+3. An autoencoder (which is just a chain of the encoder and decoder)
 
 The only variable we give our model is the encoding dimensions, which will be a hyperparemter of our final transformer.
 
-```python
-from tensorflow import keras
+We will build these models within `_keras-build_fn` so that we can store our encoders and decoders as seperate models and call them from `transform` and `inverse_transform` respectively.
```

Comment:
```suggestion
We will build these models within `_keras_build_fn` so that we can store our encoders and decoders as seperate models and call them from `transform` and `inverse_transform` respectively.
```

If `_keras_build_fn` is mentioned in the documentation, should it be private?

Reply:
It could be made public. But generally I think of making a method public when it needs to be called from outside the class. This only ever called from fit, so to me it makes sense to keep it private.

Reply:
What determines if a function needs to be called? Why doesn't this usage qualify?

My metric for public/private "will users every call this function?" I think if a function is shown in an example on the documentation page, then that function should be public.

Reply:
I do think there's good arguments for making it public. I opened #202 to track this. I don't think this should block this PR. `_keras_build_fn` is mentioned and used elsewhere in the docs, so we can't resolve this here.